README.md

```diff
@@ -101,6 +101,13 @@ __Note:__ Although packages were previously offered on certain
 GCM no longer publishes to these repositories. Please install the
 Debian package using the above instructions instead.
 
+To uninstall:
+
+```shell
+git-credential-manager-core unconfigure
+sudo dpkg -r gcmcore
```

Comment:
cc @ldennington this may need updating depending on the ordering of the "rename" PR

Reply:
Mmm? I don't think I actually ended up changing the application names. Or are you referencing the future PR where that will happen?